### ISSUE TITLE
theme skin

### ISSUE THREAD

**Issue by katsu1110** [academicpages/academicpages.github.io#137]
Thanks for a great repos. Is there a way to change the color skin (e.g. minimal_mistakes_skin) from the default?

**Comment by katsu1110** [comment]
I kind of made it work, by disabling 'assets' by putting 'assets' into the exclude list in the _config.yml.
If you do it, you can change the skin of your website by putting the following on top of everything in the _config.yml

plugins:
  - jekyll-remote-theme
remote_theme           : "mmistakes/minimal-mistakes"
minimal_mistakes_skin    : "dark" # "air", "aqua", "contrast", "dark", "dirt", "neon", "mint", "plum", "sunrise"

**Comment by katsu1110** [comment]
Oh yes, sure

simply add the following in the beginning of your "_config.yml"

`
plugins:
  - jekyll-remote-theme
remote_theme           : "mmistakes/minimal-mistakes"
minimal_mistakes_skin    : "dark" # "air", "aqua", "contrast", "dark", "dirt", "neon", "mint", "plum", "sunrise"
`

Then just put 'assets' in the list of 'exclude' in the "_config.yml" to disable all the css.

`
# Reading Files
include:
  - .htaccess
  - _pages
  - files
exclude:
  - "*.sublime-project"
  - "*.sublime-workspace"
  - .asset-cache
  - .bundle
  - .jekyll-assets-cache
  - .sass-cache
  - assets
`

Once you push it, your website should have a 'dark' skin with little loss of the format:)

**Comment by anebz** [comment]
Tried it, the skin is indeed dark but the sidebar is pushed all the way to the left and the navbar on top is slightly disfigured, @katsu1110 is this what you mean by little loss of the format? :) or is there something I'm missing?

**Comment by katsu1110** [comment]
@anebz  Exactly, this is what I meant. For this academic pages, it seems that there are some unique changes in css from the original minimal_mistakes, which prevent us from changing the theme easily.

**Comment by nicklalone** [comment]
It's a shame it seems to remove all iconography on the profile page. I wonder which asset specifically is blocking the background content.

**Comment by anebz** [comment]
Exactly, especially since it's supposed to be a simple one line addition, according to the documentation.

**Comment by eisenjulian** [comment]
Was anyone able to make this work?

**Comment by anebz** [comment]
I wasn't :(

**Comment by datasumowrestler** [comment]
Had the same problem. This worked for me:
[https://github.com/mmistakes/minimal-mistakes/issues/1339](url)

In _config.yml, add/ edit the following:
`url: https://vg.github.io`

**Comment by bensdm** [comment]
can you detail please?

**Comment by mrsandeshbhat** [comment]
I was able to change the theme of my academic pages website using the color values listed in the _sass/skins/dark.scss file into the _variables.css file as can be seen [here](https://mrsandeshbhat.github.io/)

my _variables.scss file can be viewed [here](https://github.com/mrsandeshbhat/mrsandeshbhat.github.io/blob/master/_sass/_variables.scss)

**Comment by anebz** [comment]
Thanks so much @mrsandeshbhat ! It worked 😎

**Comment by steipatr** [comment]
> I was able to change the theme of my academic pages website using the color values listed in the _sass/skins/dark.scss file into the _variables.css file as can be seen [here](https://mrsandeshbhat.github.io/)
> 
> my _variables.scss file can be viewed [here](https://github.com/mrsandeshbhat/mrsandeshbhat.github.io/blob/master/_sass/_variables.scss)

@mrsandeshbhat, Are any further changes needed to propagate these color value changes? I tried editing my _variables.scss but the website colors appear unchanged.

**Comment by anebz** [comment]
> 
> 
> > I was able to change the theme of my academic pages website using the color values listed in the _sass/skins/dark.scss file into the _variables.css file as can be seen [here](https://mrsandeshbhat.github.io/)
> > my _variables.scss file can be viewed [here](https://github.com/mrsandeshbhat/mrsandeshbhat.github.io/blob/master/_sass/_variables.scss)
> 
> @mrsandeshbhat, Are any further changes needed to propagate these color value changes? I tried editing my _variables.scss but the website colors appear unchanged.

@steipatr I tried doing [these changes](https://github.com/anebz/anebz.github.io/commit/07c9a7701fb1fb8166e83a8d83351fd09b2f309f) and it worked

**Comment by steipatr** [comment]
@anebz Thanks! I copied your the contents of your _variables.scss into mine, but nothing changed whatsoever. So bizarre. Are the skins you created also needed? That seems unintuitive, since they are not activated anywhere.

**Comment by anebz** [comment]
@steipatr yeah the skins aren't referenced anywhere, I don't understand much CSS I just added the skins and it works for me. DId you manage to make it work without the skins?

**Comment by steipatr** [comment]
@anebz I'm a complete CSS noob. Couldn't get any color changes to work with the academicpages template. Trying minimal-mistakes now and it works as expected there, so trying to get a better understanding of how the different files work together in that.

**Comment by pritishmishra** [comment]
@steipatr Clone your repo to local. Remove `Gemfile.lock` file and check. Run the command `bundle exec jekyll liveserve` as given in [Readme.](https://github.com/academicpages/academicpages.github.io#to-run-locally-not-on-github-pages-to-serve-on-your-own-computer) 
Push your changes to your github repo now.
Visit your webpage now and it should reflect the changes.

**Comment by eashanadhikarla** [comment]
I want to try out smaller fontsize for the entire website. I am not sure which one to edit. Can someone point me out the right place. Thanks!

**Comment by ryo-ARAKI** [comment]
> > I was able to change the theme of my academic pages website using the color values listed in the _sass/skins/dark.scss file into the _variables.css file as can be seen [here](https://mrsandeshbhat.github.io/)
> > my _variables.scss file can be viewed [here](https://github.com/mrsandeshbhat/mrsandeshbhat.github.io/blob/master/_sass/_variables.scss)
> 
> @mrsandeshbhat, Are any further changes needed to propagate these color value changes? I tried editing my _variables.scss but the website colors appear unchanged.

I have been suffered by the exact same problem as @steipatr mentioned.
I did implement the changes @anebz shared with us and nothing happened on the local build & after `git push` ed to the remote server.
However, the next morning I found the dark theme is successfully applied.
I am completely new to CSS so have no idea what is going on in the background, but I would suggest you wait for a couple of days if you have experienced this kind of weird behavior.

**Comment by menisadi** [comment]
> I was able to change the theme of my academic pages website using the color values listed in the _sass/skins/dark.scss file into the _variables.css file as can be seen [here](https://mrsandeshbhat.github.io/)
> 
> my _variables.scss file can be viewed [here](https://github.com/mrsandeshbhat/mrsandeshbhat.github.io/blob/master/_sass/_variables.scss)

Tried and it worked! Thanks!
Only issue: the footer (with the copyright text) is still white for some reason. Any idea how to make it dark also?

**Comment by angadhn** [comment]
> > I was able to change the theme of my academic pages website using the color values listed in the _sass/skins/dark.scss file into the _variables.css file as can be seen [here](https://mrsandeshbhat.github.io/)
> > my _variables.scss file can be viewed [here](https://github.com/mrsandeshbhat/mrsandeshbhat.github.io/blob/master/_sass/_variables.scss)
> 
> Tried and it worked! Thanks! Only issue: the footer (with the copyright text) is still white for some reason. Any idea how to make it dark also?

As was done in _variables.css, you need to go in and manually change this in _sass/footer.scss. My solution is [here](https://github.com/angadhn/angadhn.github.io/commit/31aa5d48c92dae11d8b1d8d29e351da0f768dcaf).

**Comment by g-h-chen** [comment]
> plugins:
>   - jekyll-remote-theme
> remote_theme           : "mmistakes/minimal-mistakes"
> minimal_mistakes_skin    : "dark" # "air", "aqua", "contrast", "dark", "dirt", "neon", "mint", "plum", "sunrise"

This works for me! The thing is that you need to wait for 10~15 min before the server can sync all the changes (I guess).

**Comment by ashasoroushpoor** [comment]
the solution works fine but whatever I change in tables.css and notices.css their header is all white in color.

**Comment by inspirepassion** [comment]
> I was able to change the theme of my academic pages website using the color values listed in the _sass/skins/dark.scss file into the _variables.css file as can be seen [here](https://mrsandeshbhat.github.io/)
> 
> my _variables.scss file can be viewed [here](https://github.com/mrsandeshbhat/mrsandeshbhat.github.io/blob/master/_sass/_variables.scss)

One thing I encounter the issue (as shown in pic below) after I only changed the color values in _sass/ _variables.scss to my liking theme skin. Apparently the background color for the heading part is supposed to be the same as the `background_color`, however, it's always white. After some digging, I realized the masthead element of the webpage had fixed the color to white. If you encounter the same situation, in order to change that, you can go to file _sass/_masthead.scss, find below code and change the background color variable from `white` to the value of `$background-color` 
`.masthead {
  position: fixed;
  //background: white;
  background: $background-color;
  # some other codes
}
`

<img width="1092" alt="iShot_2024-07-07_03 30 28" src="https://github.com/academicpages/academicpages.github.io/assets/19159377/a8a7079f-da86-41b9-99fd-006042180866">

**Comment by zer08t7** [comment]
> > I was able to change the theme of my academic pages website using the color values listed in the _sass/skins/dark.scss file into the _variables.css file as can be seen [here](https://mrsandeshbhat.github.io/)
> > my _variables.scss file can be viewed [here](https://github.com/mrsandeshbhat/mrsandeshbhat.github.io/blob/master/_sass/_variables.scss)
> 
> I encountered the issue (as shown in the pic below) after I only changed the color values in _sass/ _variables.scss to my liking theme skin. The background color for the heading part is supposed to be the same as the `background_color`, however, it's always white. After some digging, I realized the masthead element of the webpage had fixed the color to white. If you encounter the same situation, in order to change that, you can go to file _sass/_masthead.scss, find the below code, and change the background color variable from `white` to the value of `$background-color`
> 
> ```
> .masthead {
>   position: fixed;
>   //background: white;
>   background: $background-color;
> }
> ```
> 
> <img alt="iShot_2024-07-07_03 30 28" width="1092" src="https://private-user-images.githubusercontent.com/19159377/346327808-a8a7079f-da86-41b9-99fd-006042180866.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQxNTE2ODksIm5iZiI6MTcyNDE1MTM4OSwicGF0aCI6Ii8xOTE1OTM3Ny8zNDYzMjc4MDgtYThhNzA3OWYtZGE4Ni00MWI5LTk5ZmQtMDA2MDQyMTgwODY2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIwVDEwNTYyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczZDY4MzhmNzA0ODIwNDg3NGQ5MWYzODYwYjY5YmE0MmYxY2EzZWZiOTVkNTQ0ZTdkNmRhZDQ2YjAyNTRlZjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.09iyUbKuojHs0g9YRatZsOU9-Dbd7B3ysiWa_orbE34">

From mobile, after clicking the follow button. The hover background is originally white, making it hard to read the text. As shown in my commit, by changing the code below can make the color as same as the background: https://github.com/zer08t7/zer08t7.github.io/commit/f0279de71582a17a130672da2925efe06ecddc38

**Comment by SourojitGhosh** [comment]
I wanted to only edit the background color of the masthead (similar to this [website](https://toby.li/)) and thus added some background labels in the masthead.css file. But I cannot seem to get one part of the default background to go away. Anyone knows how to fix? 
![image](https://github.com/user-attachments/assets/95bedb0b-5812-414d-9812-771d2d923835)

**Comment by rjzupkoii** [comment]
The `_sass` directory was reorganized and support for themes added. A dark mode theme has been added with [v0.8.3 release](https://github.com/academicpages/academicpages.github.io/releases/tag/v0.8.3), see [the wiki](https://github.com/academicpages/academicpages.github.io/wiki/Customizing-the-theme-of-your-site) for more details.
